src/datepicker/__tests__/datepicker-int.scenario.js

```diff
@@ -9,18 +9,14 @@ LICENSE file in the root directory of this source tree.
 import * as React from 'react';
 
 import {StatefulDatepicker} from '../index.js';
-import {hu} from 'date-fns/locale/index.js';
 
 export default function Scenario() {
   return (
     <StatefulDatepicker
       aria-label="Select a date"
-      formatString="yyyy EEEE d"
-      placeholder="dd.mm.yyyy – dd.mm.yyyy"
-      locale={hu}
-      highlightedDate={new Date('March 10, 2019')}
-      mask={null}
-      range
+      formatString="dd.MM.yyyy"
```

Comment:
How about we create a separate scenario for a formatting test case and keep the localization one as is?

Reply:
I was more leaning towards the point, that formatting is a part of internationalization. 
But if you think that would be preffered, then for sure i can move it to separate test.

Reply:
The locale is used in the formatting fn.

Reply:
Done